### PR TITLE
Update exometer to our forked version.

### PR DESCRIPTION
**Pull request by kzemek**
This PR updates exometer to a version maintained by us: [esl/exometer_core](https://github.com/esl/exometer_core), [esl/exometer_report_graphite](https://github.com/esl/exometer_report_graphite) and [esl/exometer_report_statsd](https://github.com/esl/exometer_report_statsd).

These versions have greatly reduced dependencies, especially on packages that historically proved to have problems with language and build tool updates, e.g. setup and edown. Consequently, they also work with Elixir without trouble and don't require overriding deps.

### CODE REVIEW

File: src/mongooseim.app.src [hunk 1]
```diff
@@ -16,8 +16,8 @@
                   csv,
                   cuesport,
                   exml,
-                  exometer,
                   exometer_core,
+                  exometer_report_graphite,
                   jiffy,
```
Comment (fenek) [inline src/mongooseim.app.src]: ... and `exometer_report_statsd` :)

File: doc/operation-and-maintenance/Logging-&-monitoring.md [hunk 2]
```diff
@@ -52,33 +52,24 @@ To monitor MongooseIM during load testing, we recommend the following open sourc
...
-
-* graphite
-* amqp
-* statsd
-* snmp
-* opentsdb
+### Plug-in Exometer reporters
 
+MongooseIM uses the Exometer libary for collecting the metrics.
+Exometer has many plug-in reporters that can send metrics to external services like graphite and statsd.
```
Comment (fenek) [inline doc/operation-and-maintenance/Logging-&-monitoring.md]: We should provide links here to specific repos, for reference.

**Comment by kzemek** [comment]
@fenek Added opts (as much as was documented / I know about) in repos.

File: doc/operation-and-maintenance/Logging-&-monitoring.md [hunk 3]
```diff
@@ -52,33 +52,23 @@ To monitor MongooseIM during load testing, we recommend the following open sourc
...
 
-**Note that we are using the 1.2.1 version with our patches.**
-
-Below you can find a sample configuration. 
+Below you can find a sample configuration.
 It shows setting up a reporter connecting to graphite running on localhost.
 
 You can see an additional option not listed in the Exometer docs - `mongooseim_report_interval`.
-That option sets the metrics resolution: how often Exometer gathers and sends metrics through reporters. 
+That option sets the metrics resolution: how often Exometer gathers and sends metrics through reporters.
```
Comment (goddammit) [inline doc/operation-and-maintenance/Logging-&-monitoring.md]: The option sets the metric resolution: how often Exometer gathers and sends the metrics through reporters.

The option sets the metrics' resolution: how often Exometer gathers and sends the metrics through reporters.

**Pull request merged**